src/css/components/all.scss

```diff
@@ -4,6 +4,7 @@
 @import 'billboard.scss';
 @import 'card.scss';
 @import 'confirm.scss';
+@import 'contents.scss';
```

Comment:
new contents styles apply to the overview home card and also to the overview panels in org and space views